sources/api/netdog/Cargo.toml

```
@@ -1,14 +1,19 @@
 [package]
 name = "netdog"
-version = "0.1.0"
+version = "0.1.1"
 authors = ["Ben Cressey <bcressey@amazon.com>"]
 license = "Apache-2.0 OR MIT"
 edition = "2021"
 publish = false
 # Don't rebuild crate just because of changes to README.
 exclude = ["README.md"]
 
+[features]
+default = []
+wicked = []
```

So that we can be explicit about the feature selection in `netdog.spec`, but still have CI test one of these paths:
```suggestion
default = ["networkd"]
networkd = []
wicked = []
```

But somewhere it should be an error to define both features.